components/gitpod-db/src/user-db.ts

```diff
@@ -143,6 +143,8 @@ export type PartialUserUpdate = Partial<Omit<User, "identities">> & Pick<User, "
 
 export const BUILTIN_WORKSPACE_PROBE_USER_ID = "builtin-user-workspace-probe-0000000";
 
+export const BUILTIN_WORKSPACE_USER_AGENT_SMITH = "buildin-user-agent-smith-0000000";
```

Comment:
:heart:

Reply:
Should it be `builtin` instead of `buildin`?

Reply:
nit: the first user is called `buil**t**in`. We should try to avoid future confusion and align the agent-smith user name :+1:

Reply:
Good catch!